sdk/textanalytics/ai-text-analytics/test/public/textAnalyticsClient.spec.ts

```diff
@@ -603,6 +603,25 @@ describe("[AAD] TextAnalyticsClient", function() {
           );
         }
       });
+
+      it("accepts pii categories", async function() {
```

Comment:
it will be good to also add a test where the user does PII recognition, and then uses the category output as an input for a next call.

Reply:
Done.